src/ds/ds5/ds5-auto-calibration.h

```diff
@@ -84,7 +85,7 @@ namespace librealsense
         DirectSearchCalibrationResult get_calibration_status(int timeout_ms, std::function<void(const int count)> progress_func, bool wait_for_final_results = true);
 
         std::vector<uint8_t> _curr_calibration;
-        std::shared_ptr<hw_monitor>& _hw_monitor;
+        std::shared_ptr<hw_monitor> _hw_monitor;
```

Comment:
Hard to see in the PR but can we add a check if it's empty before using it?
Then we can throw and tell the user he must call `set` function before

Reply:
We can, but the behavior in the case the hwm is empty has not changed. And no device constructor would succeed without that. 
The user will not have to add any "set" method, but only developers. I do not think this is required - let's talk about it if you want.

Reply:
NP, accepted.